pyasdf/tests/test_utils.py

```diff
@@ -93,3 +93,8 @@ def test_sizeof_fmt_function():
     assert sizeof_fmt(1024 ** 2) == "1.0 MB"
     assert sizeof_fmt(1024 ** 3) == "1.0 GB"
     assert sizeof_fmt(1024 ** 4) == "1.0 TB"
+
+
+def test_replace_None():
```

Comment:
Can you please rename this to `test_replace_none()` to follow the naming conventions.